lib/AbstractReader.js

```diff
@@ -70,6 +70,14 @@ class AbstractReader {
 		});
 	}
 
+	filter(callback) {
+		const ReaderFilter = require("./ReaderFilter");
```

Comment:
Note: Cyclic dependency
This shouldn't cause any issues but we might also just use the "ReaderFilter" directly within ui5-builder.